drivers/flash/flash_stm32_ospi.c

```diff
@@ -952,6 +947,14 @@ static int flash_stm32_ospi_erase(const struct device *dev, off_t addr,
 					: HAL_OSPI_INSTRUCTION_8_BITS;
 
 	while ((size > 0) && (ret == 0)) {
+
+		if (stm32_ospi_write_enable(&dev_data->hospi,
```

Comment:
Does the code of the flash_stm32_ospi_write() function fits here :

```suggestion
		ret = stm32_ospi_write_enable(&dev_data->hospi,
						    dev_cfg->data_mode, dev_cfg->data_rate);
		if (ret != 0) {
			LOG_ERR("Erase failed : write enable");
			break;
		```

Reply:
Thanks, I will update this